openapi/go-client-generated/docs/ExecutionResult.md

```diff
@@ -7,6 +7,7 @@ Name | Type | Description | Notes
 **BlockId** | **string** |  | [default to null]
 **Events** | [**[]Event**](Event.md) |  | [default to null]
 **Links** | [***Links**](Links.md) |  | [optional] [default to null]
+**ExecutionDataID** | **string**  |  | [default to null]
```

Comment:
It looks like this is an auto-generated file. You should probably change the definition at the source (ie. the OpenAPI spec), and then regenerate this file.

However, currently we haven't updated the REST API to have this change, so for now we can probably omit this entirely.

Reply:
ok will do